packages/ui/src/theme/css/component/fileUploader.scss

```diff
@@ -36,3 +37,48 @@
   font-size: var(--amplify-font-sizes-small);
   margin-block-start: var(--amplify-space-small);
 }
+
+.amplify-fileuploader__previewer {
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  max-height: 40rem;
+  max-width: 25rem;
+  overflow: hidden;
+}
+
+.amplify-fileuploader__previewer__body {
+  display: flex;
+  flex: 1;
+  overflow: auto;
+  flex-direction: column;
+  gap: var(--amplify-space-small);
+  padding: var(--amplify-space-medium);
+}
+
+.amplify-fileuploader--footer {
+  position: relative;
+  padding: var(--amplify-space-medium);
+  display: flex;
+  flex-direction: row;
+  border-top: var(--amplify-border-widths-small) solid
+    var(--amplify-colors-border-secondary);
+}
+
+.amplify-fileuploader--footer > .amplify-fileuploader-loader {
+  position: absolute;
```

Comment:
Is position absolute necessary?

Reply:
it is yes, I am working on cleaning up the styling code right now so I wouldn't block this PR for styling things.